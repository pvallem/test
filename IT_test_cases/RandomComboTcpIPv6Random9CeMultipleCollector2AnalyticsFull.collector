bidir_flow_info {
  key {
    src_address: "\031Z\371\354\264C;\n\325\356\262\353\033=*\032"
    dst_address: "L#\242T\341]\24328\230w\001\236v\277P"
    src_port: 57407
    dst_port: 11311
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000040
    key_type: IPV6
  }
  reverse_flow_info {
    start_time: 1388563200000040
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 20
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000040
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\002\002\002\002\002\001"
    dst_address: "\002\002\002\002\002d"
    tenant_id: 10
    flow_start_time: 1388563200000030
    key_type: MAC
    ether_type: 34827
  }
  flow_info {
    start_time: 1388563200000030
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000030
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\235*\030>-\034\027\323\351\302>\256\266c\271\301"
    dst_address: "\320U\347UTq\320\r:\303\324m$D\360P"
    src_port: 60174
    dst_port: 49883
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV6
  }
  flow_info {
    start_time: 1388563200000000
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 0
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000000
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "U#H\243c\357\255]\372\326\201\3072-\371?"
    dst_address: "\313\360v\376\254\212\031n>\320K\313\303T\223@"
    src_port: 18129
    dst_port: 5333
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: IPV6
  }
  flow_info {
    start_time: 1388563200000020
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 10
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000020
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\002\002\002\002\002\000"
    dst_address: "\002\002\002\002\002d"
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: MAC
    ether_type: 34827
  }
  flow_info {
    start_time: 1388563200000010
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000010
    sensor_id: "127.0.0.1:5675"
  }
}
