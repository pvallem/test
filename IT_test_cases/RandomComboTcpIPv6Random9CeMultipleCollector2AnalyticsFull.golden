flow_info {
  key {
    src_address: "\235*\030>-\034\027\323\351\302>\256\266c\271\301"
    dst_address: "\320U\347UTq\320\r:\303\324m$D\360P"
    src_port: 60174
    dst_port: 49883
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV6
  }
  start_time: 1388563200000000
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 0
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000000
}
flow_info {
  key {
    src_address: "\002\002\002\002\002\000"
    dst_address: "\002\002\002\002\002d"
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: MAC
    ether_type: 34827
  }
  start_time: 1388563200000010
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000010
}
flow_info {
  key {
    src_address: "U#H\243c\357\255]\372\326\201\3072-\371?"
    dst_address: "\313\360v\376\254\212\031n>\320K\313\303T\223@"
    src_port: 18129
    dst_port: 5333
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: IPV6
  }
  start_time: 1388563200000020
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 10
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000020
}
flow_info {
  key {
    src_address: "\002\002\002\002\002\001"
    dst_address: "\002\002\002\002\002d"
    tenant_id: 10
    flow_start_time: 1388563200000030
    key_type: MAC
    ether_type: 34827
  }
  start_time: 1388563200000030
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000030
}
flow_info {
  key {
    src_address: "L#\242T\341]\24328\230w\001\236v\277P"
    dst_address: "\031Z\371\354\264C;\n\325\356\262\353\033=*\032"
    src_port: 11311
    dst_port: 57407
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000040
    key_type: IPV6
  }
  start_time: 1388563200000040
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 20
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000040
}
